src/AutoRest.CSharp/LowLevel/AutoRest/DpgTarget.cs

```diff
@@ -37,12 +37,13 @@ public static async Task ExecuteAsync(GeneratedCodeWorkspace project, InputNames
             {
                 var codeWriter = new CodeWriter();
                 var xmlDocWriter = new XmlDocWriter();
-                var lowLevelClientWriter = new LowLevelClientWriter(codeWriter, xmlDocWriter, client);
+                var exampleComposer = new LowLevelExampleComposer(client, library.TypeFactory);
+                var lowLevelClientWriter = new LowLevelClientWriter(codeWriter, xmlDocWriter, client, exampleComposer);
```

Comment:
I have a PR to refactor everything around the example composer that might impact your current refactor, which you might be interested: https://github.com/Azure/autorest.csharp/pull/3672

Reply:
Resolved